source/Panel.cpp

```diff
@@ -93,7 +93,7 @@ void Panel::AddZone(const Rectangle &rect, const function<void()> &fun)
 
 void Panel::AddZone(const Rectangle &rect, SDL_Keycode key)
 {
-	AddZone(rect, [this, key](){ this->KeyDown(key, 0, Command(), true); });
+	AddZone(rect, [this, key](){ this->KeyDown(key, SDL_GetModState(), Command(), true); });
```

Comment:
Without this change, holding shift and clicking on the park buttons would not yield the proper behavior; it was only doing what was expected if I used the hotkeys.
I'm unsure of if this will cause any issues, as this is now the case for all panels it seems. I can't off the top of my head see any places where this might be an issue, but I'm sure there was a reason this ignored the mod state in the first place.